packages/data-service/src/data-service.ts

```diff
@@ -1579,7 +1579,8 @@ class DataServiceImpl extends WithLogContext implements DataService {
     debug('connecting...');
     this._isConnecting = true;
 
-    this._logger.info(mongoLogId(1_001_000_014), 'Connecting', {
+    this._logger.info(mongoLogId(1_001_000_014), 'Connecting Started', {
```

Comment:
I am inspired here by the driver's commandStarted/Succeeded/Failed. But the message changes aren't necessary so happy to undo.

Reply:
If the id stays the same and conceptually it's the same event I think that's fine (but you will need to figure out all the tests that might be checking this 🙂)